test/unit/test_events.py

```diff
@@ -1,8 +1,9 @@
 from dbt.events import AdapterLogger
 from dbt.events.types import AdapterEventDebug
+from dbt.events.base_types import Cli,File,Event
```

Comment:
Just the commas here too
```suggestion
from dbt.events.base_types import Cli, File, Event
```

Reply:
Removed unused imports